icu4c/source/common/unicode/unistr.h

```diff
@@ -327,6 +328,53 @@ class U_COMMON_API UnicodeString : public Replaceable
    */
   inline bool operator== (const UnicodeString& text) const;
 
+#ifndef U_HIDE_DRAFT_API
+  /**
+   * Equality operator. Performs only bitwise comparison with a
+   * std::u16string_view or (if U_SIZEOF_WCHAR_T==2) std::wstring_view.
+   * TODO: Should we document this as what it is, "convertible to one of these string views",
+   * rather than claiming that it needs to *be* one?
+   *
+   * @param text The string view to compare to this string.
+   * @return true if `text` contains the same characters as this one, false otherwise.
+   * @draft ICU 76
+   */
+  template<typename StringView, typename = std::enable_if_t<ConvertibleToU16StringView<StringView>>>
```

Comment:
Also, since this does not have to _be_ a string view but only has to be convertertible to one, I am tempted to change the typename StringView to simply S. As in
```
template<typename S, typename = std::enable_if_t<ConvertibleToU16StringView<S>>>
```

WDYT?

Reply:
I agree StringView is not a great name here. No objections to S (for better or worse, something like `typename T` is _very_ idiomatic when one does not have any better ideas; at least S is slightly clearer than T here).